core/tests/unit/kmx/fixtures/meson.build

```
@@ -0,0 +1,6 @@
+if node.found()
+  # Note: if node is not available, we cannot build the keyboards; build.sh
+  # emits a warning that the 'ldml' keyboard tests will be skipped; that
```

```suggestion
  # Note: if node is not available, we cannot build the binary test keyboards; build.sh
  # emits a warning that the 'kmx/binary' keyboard tests will be skipped; that
```

looks like a copy and paste…

---

Actually, at present build.sh still reports that it can't build 'ldml' keyboards 😆... low priority message which we should hopefully never see.